tests/child-wait.test.sh

```
@@ -91,8 +91,8 @@ MALFORMED_LEASE_ERROR="$(_uberdev_child_terminal_lease_proof "$STATUS" 2>&1)"
 MALFORMED_LEASE_RC=$?
 set -e
 [ "$MALFORMED_LEASE_RC" -ne 0 ]
-printf '%s\n' "$MALFORMED_LEASE_ERROR" | grep -Fq "invalid lifecycle lease: $MALFORMED_LEASE_REAL"
-! printf '%s\n' "$MALFORMED_LEASE_ERROR" | grep -Fq 'generation=bad'
+grep -Fq "invalid lifecycle lease: $MALFORMED_LEASE_REAL" <<<"$MALFORMED_LEASE_ERROR"
+! grep -Fq 'generation=bad' <<<"$MALFORMED_LEASE_ERROR"
```

Shellcheck (0.11.0)

[info] This ! is not on a condition and skips errexit. Use `&& exit 1` instead, or make sure $? is checked.

(SC2251)